components/vc/front/src/torque.c

```diff
@@ -120,6 +121,15 @@ static void torque_periodic_100Hz(void)
         }
     }
 
+#if FEATURE_IS_ENABLED(FEATURE_REVERSE)
```

Comment:
god I hate that you have to use `FEATURE_IS_ENABLED`... I need to look through the code for this feature tree stuff. If you just define these as `#define FTR_REVERSE true` in your codegen, you can then just do `#if FTR_REVERSE`

Reply:
_Technically_ the feature is defined as 1 so it will correctly evaluate. Thought itd be better to be verbose and allow `FEATURE_IS_ENABLED(...)` and `FEATURE_IS_DISABLED(...)`

Reply:
I originally kept it seperate to distinguish from ENABLED features, and "FEATURES" that are enums (see APP_COMPONENT_ID iirc)